netmanager/src/views/pages/Dashboard/components/AveragesChart.js

```diff
@@ -397,7 +405,13 @@ const AveragesChart = ({ classes }) => {
         <Divider />
         <CardContent>
           <div className={classes.chartContainer}>
+            {loading ? (
+              <div>
```

Comment:
@Codebmk Could we add some css (height and width) to ensure the shape  of the chart containers are not reduced when the loader is one

Reply:
@ja-odur I am unable to see the containers shrink even though for me initially they were instead increasing in size because of the styles i was applying to the loader.

Reply:
I also tried using flexbox to center the loader but things have failed😅

Reply:
@ja-odur I have added height to the loader container. Please review